### ISSUE TITLE
Documenting failure cases

### ISSUE THREAD

**Issue by LPardue** [DavidSchinazi/draft-schinazi-connect-udp-listen#4]
I can imagine some failure cases for this extension that might benefit from some explicit documentation. 

This document defers its security considerations to CONNECT-UDP and that is good. But, for example, where CONNECT-UDP says something like 

> [some IP ranges are special, clients might try them and] UDP proxies can use the
   destination_ip_prohibited Proxy Error Type from Section 2.3.5 of
   [[PROXY-STATUS](https://datatracker.ietf.org/doc/html/draft-ietf-masque-connect-udp-15#ref-PROXY-STATUS)] when rejecting such requests.

there's not a direct equivalent in this document because the request for `*` can't be be enforced in the same way. Instead, what might happen is a client or proxy could include a bad IP address like 127.0.0.1 by including it in the datagram payload. The recipient of this datagram would probably want to quietly drop that datagram, or maybe go so far as resetting the stream, or perhaps something in between (drop and signal?).

**Comment by DavidSchinazi** [comment]
Good point, we should at the very least add some recommendations so implementers know what options they have
